optional/radicale/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM alpine:edge
+ARG EDGE_DISTRO=alpine:edge
+FROM $EDGE_DISTRO
 
 RUN echo "@testing http://nl.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories \
- && apk add --no-cache radicale@testing py-dulwich@testing curl bash
+  && apk add --no-cache py3-dulwich@testing curl bash
+RUN pip3 install radicale
 
```

Comment:
Why drop official distro package for pip one? This change should not be related to ARM. If it is an improvement by itself, please open a specific pull request.